### PR TITLE
Fix mouse scratch after waveform reset. 

### PR DESCRIPTION
**Pull request by daschuer**
It feels interesting to only rely on ears when verifying the beat position marker.

### CODE REVIEW

File: src/waveform/renderers/waveformwidgetrenderer.cpp [hunk 1]
```diff
@@ -123,18 +124,20 @@ void WaveformWidgetRenderer::onPreRender(VSyncThread* vsyncThread) {
...
+        if (pWaveform) {
+            m_audioVisualRatio = pWaveform->getAudioVisualRatio();
+        }
     }
 
+    m_audioSamplePerPixel = m_visualSamplePerPixel * m_audioVisualRatio;
+
     double truePlayPos = m_visualPlayPosition->getAtNextVSync(vsyncThread);
-    // m_playPos = -1 happens, when a new track is in buffer but m_visualPlayPosition was not updated
+    // ruePlayPos = -1 happens, when a new track is in buffer but m_visualPlayPosition was not updated
```
Comment (ronso0) [inline src/waveform/renderers/waveformwidgetrenderer.cpp:139]: typo truePlayPos

**Comment by ronso0** [comment]
which situation exactly is fixed by this PR?

**Comment by daschuer** [comment]
Try to mouse scratch the waveform after resetting the waveform.

File: src/waveform/renderers/waveformwidgetrenderer.cpp (hunk 1 above)
Comment (daschuer) [reply src/waveform/renderers/waveformwidgetrenderer.cpp:139]: I have fixed it by a squash commit. I hope it is OK in this case.

Comment (ronso0) [reply src/waveform/renderers/waveformwidgetrenderer.cpp:139]: sure :)

**Comment by ronso0** [comment]
Thanks, works as expected. LGTM

**Pull request merged**